shapeways/oauth2_client.py

```diff
@@ -47,6 +48,51 @@ def authenticate(self, client_id, client_secret):
         print(response.content)
         return False
 
+    # Oauth2 authentication method
+    def authenticate_authorization_code(self, client_id, client_secret, authorizaion_code='eBGI9tpcMrBZwD9Yguqz'):
```

Comment:
I'd suggest removing the `authorization_code`'s default value since the code is required by the request and quickly expires (default value becomes obsolete).

Reply:
I agree.